Source/Blazorise.Bootstrap/wwwroot/modal.js

```diff
@@ -116,4 +116,4 @@ export function resetAdjustments(element) {
 export function getScrollBarWidth() {
     const documentWidth = document.documentElement.clientWidth;
     return Math.abs(window.innerWidth - documentWidth);
-}
\ No newline at end of file
+}
```

Comment:
There are a bunch of changes in JS files that seem to be because of line endings. Can we revert them to the previous version?

Reply:
Lol, honestly no idea we just replace with `File.ReadAllText(fileName)` I'll try to figure it out.

Reply:
added **.trim** Seems to have fixed it.